full-service/src/json_rpc/json_rpc_request.rs

```diff
@@ -66,11 +66,12 @@ pub enum JsonCommandRequest {
     },
     build_and_submit_transaction {
         account_id: String,
-        addresses_and_values: Option<Vec<(String, String)>>,
+        addresses_and_values: Option<Vec<(String, String, String)>>,
         recipient_public_address: Option<String>,
-        value_pmob: Option<String>,
+        value: Option<(String, String)>,
```

Comment:
Can we express Token Type explicitly in the API? I think this proposed API is confusing and error prone, as it's easy to put the wrong string in the wrong index, and it's not deducible what each value represents, or the constraints on that value.